### PR TITLE
I totally have no idea what im doing

### PR DESCRIPTION
**Pull request by mishl-dev**
I hate the bot splits responses

### CODE REVIEW

**Comment by eris-webserv** [comment]
considering the edits to .gitignore I have to assume you tried to run the bot using node. this is a bot coded EXCLUSIVELY for Deno.

it uses deno APIs. it uses deno features. trying to run it in node (or Bun) will cause it to violently implode and stop functioning.

will review later

**Comment by eris-webserv** [comment]
> I hate the bot splits responses

how do you think I handle the LLM giving responses over 2000 characters without just... cutting them off

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,71 @@
...
+yarn.lock
+yarn-error.log
+.parallelperf.*
+tests/baselines/reference/dt
+.failed-tests
+TEST-results.xml
+package-lock.json
+.eslintcache
+*v8.log
+/lib/
```
Comment (eris-webserv) [inline .gitignore:71]: what, why?? were you trying to run this with node??

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ DISCORD_TOKEN=""
 OPENAI_API_KEY=""
 BING_COOKIE=""
 GEMINI_API_KEY=""
-DISCORD_APP_ID=""
+APP_ID=""
```
Comment (eris-webserv) [inline README.md:15]: oh shit I never did update the .env with this did I. good catch.

**Review by eris-webserv — COMMENTED** [review]
oh no

**Review by eris-webserv — APPROVED** [review]
after violating the laws of Programming:tm: I have tested this change in production and it has worked.

I'm going to nuke your .gitignore changes though.

File: .gitignore (hunk 1 above)
Comment (mishl-dev) [reply .gitignore:71]: i just copypasted a generic javascript gitignore

Comment (eris-webserv) [reply .gitignore:71]: no need, the gitignore LLM Bot uses is sufficient. Deno doesn't do a lot of node's stuff.

File: README.md (hunk 2 above)
Comment (mishl-dev) [reply README.md:15]: common skill issue

**Pull request merged**